pkg/kp/statusstore/podstatus/status.go

```diff
@@ -23,6 +24,10 @@ const (
 	Pending  ServiceState = "pending"
 	Running  ServiceState = "running"
 	Finished ServiceState = "finished"
+
+	PodLaunched PodState = "launched"
```

Comment:
I'm hoping that the types being different (`PodState` vs `ServiceState`) will prevent this. You're right though that one could be unmarshaled into the wrong type though. Should I modify the strings e.g. `pod_launched` `pod_pending`?